modules/nf-core/cellranger/mkvdjref/main.nf

```diff
@@ -0,0 +1,39 @@
+process CELLRANGER_MKVDJREF {
+    tag "$fasta"
+    label 'process_high'
+
+    container "nfcore/cellranger:7.1.0"
+
+    // Exit if running this module with -profile conda / -profile mamba
+    if (workflow.profile.tokenize(',').intersect(['conda', 'mamba']).size() >= 1) {
+        exit 1, "CELLRANGER_MKREF module does not support Conda. Please use Docker / Singularity / Podman instead."
+    }
+
+    input:
+    path fasta
+    path gtf
+    val reference_name
```

Comment:
Can you confirm all mandaotry but also optional inputs are covered here (not familiar with the tool)

Reply:
yes, any extra arguments are passed to `cellranger` via `task.ext.args`